packages/frontend/web/components/MostViewed.tsx

```diff
@@ -303,135 +331,176 @@ export class MostViewed extends Component<Props, { selectedTabIndex: number }> {
     }
 
     public render() {
+        const { config } = this.props;
         return (
-            <div
-                className={container}
-                data-link-name={'most-viewed'}
-                data-component={'most-viewed'}
-            >
-                <h2 className={heading}>Most popular</h2>
-                <div className={mostPopularBody}>
-                    <AsyncClientComponent f={this.fetchTrails}>
-                        {({ data }) => (
-                            <div className={listContainer}>
-                                {Array.isArray(data) && data.length > 1 && (
-                                    <ul
-                                        className={tabsContainer}
-                                        role="tablist"
-                                    >
+            <div className={`content-footer ${cx(adSlotUnspecifiedWidth)}`}>
+                <OutbrainContainer config={config} />
+                <ArticleContainer showTopBorder={true}>
```

Comment:
This is a little confusing here. `ArticleContainer` to me implies that this will contain the article (content). However, it's under Outbrain! Looking at the component, it's kind of a styling provider? I can see it came from using the same styles in two places in the original component.

My feeling is that it might be clearer to remove the HOC completely and duplicate some of the style/logic in the two places? What do you think?

Reply:
You're right, it's confusing. Changed. I think there're still are some wins we could get here but maybe by editing Container directly. One for later.